### PR TITLE
App sharing feature Enhancement

### PR DESCRIPTION
**Pull request by NoahNsimbe**
#### Summary of Changes (What does this PR do?)
- Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

#### Is this change ready to hit production in its current state?
- [x] Yes
- [ ] No

#### Status of maturity (all need to be checked before merging):

- [x] I consider this code done
- [x] I've tested this locally
- [ ] This branch has been LGTM'ed by a reviewer

#### How should this be manually tested?
* Please include the steps to be done inorder to setup and test this PR.

#### What are the relevant tickets?
- [ticket_id]()

#### Screenshots (optional)

### CODE REVIEW

**Review by Baalmart — COMMENTED** [review]
The files changed are quite many! Makes it impossible to review the PR. Could it be that the PR is mostly about file structural changes?  @NoahNsimbe 

Could you add some review comments to the PR, just to continuously "onboard" the rest of the team.

**Comment by NoahNsimbe** [comment]
> The files changed are quite many! Makes it impossible to review the PR. Could it be that the PR is mostly about file structural changes? @NoahNsimbe
> 
> Could you add some review comments to the PR, just to continuously "onboard" the rest of the team.

@Baalmart only three files have been changed.

File: mobile/lib/services/native_api.dart [hunk 1]
```diff
@@ -691,6 +714,10 @@ class ShareService {
     // }
   }
 
+  String getShareMessage(){
+    return 'Download the AirQo app from Google play\nhttps://play.google.com/store/apps/details?id=com.airqo.app\nand App Store\nhttps://itunes.apple.com/ug/app/airqo-monitoring-air-quality/id1337573091\n';
+  }
```
Comment (NoahNsimbe) [inline mobile/lib/services/native_api.dart:719]: Adding links to app store and play store

**Review by 123MwanjeMike — APPROVED** [review]

**Review by Baalmart — APPROVED** [review]
LGTM

**Pull request merged**